### PR TITLE
feat: add galxe data provider

### PR DESCRIPTION
**Pull request by sampolgar**
adding galxe data provider and group generator as per https://github.com/sismo-core/sismo-hub/issues/1424

### CODE REVIEW

**Review by MartinGbz — COMMENTED** [review]
Hey @sampolgar I let you 2 small comments

**Comment by sampolgar** [comment]
Thanks @MartinGbz!

File: group-generators/helpers/data-providers/index.ts [hunk 1]
```diff
@@ -97,6 +100,8 @@ export const dataProvidersInterfacesSchemas: DataProviderInterface[] = [
   attestationStationInterfaceSchema,
   ankrInterfaceSchema,
   degenScoreInterfaceSchema,
+  duneInterfaceSchema,
```
Comment (MartinGbz) [inline group-generators/helpers/data-providers/index.ts:103]: can you remove `duneInterfaceSchema` (that's the reason why it cause a problem in the CI)

**Review by MartinGbz — COMMENTED** [review]
one last comment ;)

File: group-generators/helpers/data-providers/index.ts (hunk 1 above)
Comment (sampolgar) [reply group-generators/helpers/data-providers/index.ts]: Thanks @MartinGbz, that's done!

**Pull request merged**